frontend/app/src/components/accounts/AccountBalanceTable.vue

```diff
@@ -205,10 +206,8 @@
   return get(collapsedXpubs)
     .filter(({ tags }) => get(visibleTags).every(tag => tags.includes(tag)))
     .reduce(
-      (previousValue, currentValue) => ({
-        amount: previousValue.amount.plus(currentValue.balance.amount),
-        usdValue: previousValue.usdValue.plus(currentValue.balance.usdValue)
-      }),
+      (previousValue, currentValue) =>
+        balanceSum(previousValue, currentValue.balance as Balance),
```

Comment:
if you change the declaration to `const collapsedXpubs: Ref<XpubAccountWithBalance[]> = ref([]);` it seems that cast is redundant.

I know we can't avoid casting completely but still we should be careful with their usage since they can hide bugs.